3rdparty/openenclave/ert.patch

```diff
@@ -665,22 +725,38 @@ index cf814d36a..5de0c930d 100644
 +        oe_sgx_enclave_properties_t* const props_payload =
 +        (oe_sgx_enclave_properties_t*)(oeimage_payload.elf.image_base + oeimage_payload.elf.oeinfo_rva);
 +        memset(props_payload->sigstruct, 0, sizeof(props_payload->sigstruct));
++
++        // get payload data offset and size from EDG-extended oeinfo section
++        const uint64_t* const p =
++            (uint64_t*)(oeimage_payload.elf.image_base + oeimage_payload.elf.oeinfo_rva + 0x800);
```

Comment:
Not sure if more EDG extensions are to follow, but maybe let's define these offsets somewhere instead of hardcoding offsets?